NewHorizons/Builder/General/SpawnPointBuilder.cs

```diff
@@ -12,54 +13,62 @@ public static class SpawnPointBuilder
         public static SpawnPoint Make(GameObject planetGO, SpawnModule module, OWRigidbody owRigidBody)
         {
             SpawnPoint playerSpawn = null;
-            if (!Main.Instance.IsWarpingFromVessel && !Main.Instance.IsWarpingFromShip && module.playerSpawnPoint != null)
+
+            var playerSpawnInfo = module.playerSpawn;
+            var shipSpawnInfo = module.shipSpawn;
+
+            // Backwards compatibility
```

Comment:
All other backwards compatibility has been handled by the Migrate method in PlanetConfig. Is there a specific reason why this is here instead?